### ISSUE TITLE
[4.1.0-unstable][ES] xbox360 controller disabled in game when bluetooth PS3 controller paired

### ISSUE THREAD

**Issue by garnieretienne** [recalbox/recalbox-os#1198]
### What happened

I've an XBOX360 controller (NOT wireless) that works correctly in ES and ingame. When I pair my PS3 wireless controller (became player 2), the XBOX360 works correctly in ES but once any game is started, it does not respond at all (cannot play, cannot quit). The PS3 controller continue to works correctly ingame.

### What should have happened

The XBOX360 should continue to be player 1 and works the same way ingame that in ES.

### Enumerate your devices

* RPI3 on SDCard
* XBOX360 controller (NOT wireless)
* Official PS3 controller (wireless)

### How to reproduce the bug

Start the raspberry with the USB XBOX360 controller connected, once in ES, pair the PS3 controller using the Playstation button and verify it's working too. XBOX controller should be player 1 and the PS3 controller player 2. Start anygame using the XBOX controller, and try to play with it or exit the game/emulator. That's what not working for me.

**Comment by garnieretienne** [comment]
Possibly reported on the forum 2 years ago? https://forum.recalbox.com/topic/858/wired-xbox360-controller-not-working-in-emulators

**Comment by substring** [comment]
Nope, 4.1 has a different xbox driver, it could work. But i'm not sure any of the devs has both ps3 and x360 pads

**Comment by twain28** [comment]
I can confirm this....I have the same issue!
My setup is:
- Latest Recalbox 4.1
- Speedlink Torid xbox 360 controller (with dongle) - PLAYER 1
- PS3 wireless controller - PLAYER 2

As soon as I paired the PS3 controller via bluetooth, the xbox360 one stopped working ingame (tried on N64). The PS3 controller works fine ingame, but I guess this setup could prove bad for multiplayer, am I right? (I still have not tried multiplayer games...)
The xbox 360 controller still works fine on main menu.

**Comment by twain28** [comment]
Just a few cents in...I read this:
https://github.com/recalbox/recalbox-buildroot/pull/477#issuecomment-247544478

Il looks like, to avoid issues, BT devices should be re-paired on 4.1...off to check this tonight...

**Comment by twain28** [comment]
Naah, this is simply a matter of bossy driver. :)
Two players work fine, as long as the start button (and the main button) are pressed by the PS3 controller.
